spring-cloud-azure-samples/spring-cloud-azure-sql-sample/src/main/java/com/example/SqlApplication.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.example;
```

Comment:
It may not be important, do we need to change this `com.example` package name into a more concrete package name?

Reply:
@warrenzhu25 agree with @sophiaso  let's have a better name other than com.example

Reply:
In my opinion, the code in this context is really an example. So package name shows the purpose. Or do you have some better suggestions?